back-end/lms/models/student.py

```diff
@@ -17,22 +49,47 @@ class Meta:
 
 class Program(models.Model):
     code = models.CharField(primary_key=True, max_length=8)
-    program = models.CharField(max_length=500, blank=True, null=True)
+    program = models.CharField(max_length=128, blank=True, null=True)
+    faculty = models.ForeignKey(Faculty, models.DO_NOTHING, blank=True)
 
     def __str__(self):
         return f'({str(self.code)}) {str(self.program)}'
 
     class Meta:
         verbose_name = 'Educational Program'
         verbose_name_plural = 'Educational Programs'
+        
 
-
-class Student(Person):
+class Student(Personel):
+    #militrary
     milgroup = models.ForeignKey(Milgroup, models.DO_NOTHING)
-    birthdate = models.DateField()
+    commissariat_city = models.CharField(max_length=32)
+    commissariat_district = models.CharField(max_length=32)
```

Comment:
Are you sure that 32 characters is enough?

Reply:
@gakhromov Give me please the name of city with the length more than 32